Resources/Prototypes/Recipes/Construction/Graphs/tables.yml

```diff
@@ -0,0 +1,81 @@
+﻿- type: constructionGraph
+  id: Tables
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: TableFrame
+          completed:
+            - !type:SnapToGrid { }
+          steps:
+            - material: MetalRod
+              amount: 2
+              doAfter: 1
+        - to: PokerTable
+          steps:
+            - material: Wood
+              amount: 2
+              doAfter: 1
```

Comment:
I don't really like this edge... Why don't you make an edge from the wood table node to the poker table node?